### PR TITLE
add version to the CLI

### PR DESCRIPTION
**Pull request by Avni-Sharma**

### CODE REVIEW

File: operatorcourier/cli.py [hunk 1]
```diff
@@ -55,11 +55,17 @@ def push(self):
...
     def nest(self):
         parser = argparse.ArgumentParser(description='Take a flat bundle directory and version nest it to eventually create an operator-registry image.')
         parser.add_argument('source_dir', help='Path of your directory of yaml files to bundle.')
         parser.add_argument('registry_dir', help='Path of your directory to be populated. If directory does not exist, it will be created.')
 
         args, leftovers = parser.parse_known_args(sys.argv[2:])
         api.nest(args.source_dir, args.registry_dir)
+
+    #Parse the version command
+    def version(self):
```
Comment (scoheb) [inline operatorcourier/cli.py]: I fail to see where the version number is printed to screen?

Comment (kevinrizza) [reply operatorcourier/cli.py]: Same. I don't think this pr actually implements the command.

Comment (awgreene) [reply operatorcourier/cli.py]: @Avni-Sharma could you please print the version number to the screen?

**Comment by kevinrizza** [comment]
This was addressed here: https://github.com/operator-framework/operator-courier/pull/44

Closing this pr as no longer relevant.

**Pull request closed without merging**